docs/team/tanyutao544.md

```diff
@@ -11,20 +11,37 @@ SoC InternApply - SoC InternApply helps CS students to track internship applicat
 
 Given below are my contributions to the project.
 
-* **New Feature**:
+###New Feature Developed:
```

Comment:
```suggestion
### New Feature Developed:
```
I believe by markdown syntax, headings require a space between the "###" and the heading name.